howso/direct/client.py

```diff
@@ -516,56 +513,64 @@ def create_trainee(  # noqa: C901
                 f"Possible causes - Howso couldn't find core "
                 f"binaries/camls or {trainee_id} trainee already exists.")
 
-        metadata = {
-            'name': trainee.name,
-            'metadata': trainee.metadata,
-            'persistence': trainee.persistence,
-        }
+        metadata = dict(
+            name=name,
+            persistence=persistence,
+            **(metadata or {})
```

Comment:
User metadata should remain a subkey here instead of unpacking.

Reply:
Addressed by: c28e6de8dded39f04ba3206735fe885f931d8d2e

Reply:
Metadata is still being passed through incorrectly. Will need to use a different variable name internal to this method so it doesn't override the incoming metadata variable. What comes in for metadata to the method should be what is returned from the method. As is, this will now return the name and persistence twice.